web-client/integration-tests/docketClerkServesNonMultiDocketableEntryOnConsolidatedCase.test.js

```diff
@@ -0,0 +1,93 @@
+import {
+  OBJECTIONS_OPTIONS_MAP,
+  SINGLE_DOCKET_RECORD_ONLY_EVENT_CODES,
+} from '../../shared/src/business/entities/EntityConstants';
+import { confirmInitiatePaperFilingServiceModalHelper } from '../src/presenter/computeds/confirmInitiatePaperFilingServiceModalHelper';
+import { docketClerkAddsPaperFiledDocketEntryAndSavesForLater } from './journey/docketClerkAddsPaperFiledDocketEntryAndSavesForLater';
+import { docketClerkConsolidatesCases } from './journey/docketClerkConsolidatesCases';
+import { docketClerkOpensCaseConsolidateModal } from './journey/docketClerkOpensCaseConsolidateModal';
+import { docketClerkSearchesForCaseToConsolidateWith } from './journey/docketClerkSearchesForCaseToConsolidateWith';
+import { docketClerkUpdatesCaseStatusToReadyForTrial } from './journey/docketClerkUpdatesCaseStatusToReadyForTrial';
+import { fakeFile } from '../integration-tests-public/helpers';
+import { loginAs, setupTest, uploadPetition } from './helpers';
+import { runCompute } from 'cerebral/test';
+import { withAppContextDecorator } from '../src/withAppContext';
+
+describe('Docket Clerk serves non multi-docketable entry on consolidated case', () => {
+  const cerebralTest = setupTest();
+
+  beforeAll(() => {
+    jest.setTimeout(30000);
+  });
+
+  afterAll(() => {
+    cerebralTest.closeSocket();
+  });
+
+  it('login as a petitioner and create the lead case', async () => {
+    const caseDetail = await uploadPetition(cerebralTest);
+    expect(caseDetail.docketNumber).toBeDefined();
+    cerebralTest.docketNumber = cerebralTest.leadDocketNumber =
+      caseDetail.docketNumber;
+  });
+
+  loginAs(cerebralTest, 'docketclerk@example.com');
+  docketClerkUpdatesCaseStatusToReadyForTrial(cerebralTest);
+  const documentFormValues = {
+    dateReceivedDay: 1,
```

Comment:
I think we need to serve the petition before a case should be ready for trial. 🤔  Is this suggesting there's an edge case bug if this test passes?